versioned_docs/version-1.30/reference/okteto-manifest.mdx

```diff
@@ -285,6 +285,13 @@ deploy:
 - `driver`: Specifies the backend to divert requests. Use `nginx` (or leave this field unspecified) to use the default Nginx driver.
 - `namespace`: The namespace that holds the full shareable environment. When a request is directed to a service that wasn't created by the `deploy` command, Okteto will automatically direct it to the environment running on this namespace.
 
+When `divert` is enabled, Okteto injects the header `baggage.okteto-divert` when a request goes through the developer's ingress. Okteto uses this to route the request between services on your personal namespaces and the shared environment. 
```

Comment:
@codyjlandstrom @ifbyol I'd like to document the header now that this is available on 1.30.1. Once we agree on the text I'll copy it to the other version.